.github/pull_request_template.md

```diff
@@ -32,6 +32,10 @@ Closes #
 
 <!-- Describe any specific details to help reviewers test or review this Pull Request -->
 
+- [ ] Integration tests passed at github/github (Optional and only available for GitHub staff)
```

Comment:
Just wanted to suggest an alternative to see how it landed, goal was to see how this could be a checklist item and if it could capture relevant info for people who hadn't seen it before:

```suggestion
- [ ] (Maintainers) Integration tests pass at github/github ([Learn more](https://github.com/github/primer-engineering/blob/main/how-we-work/testing-primer-react-pr-at-dotcom.md))
```

Reply:
This looks good - I think we would only need to update the "Learn more" link text to avoid generic link text (https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)